docs/source/_json/breadcrumbs.json

```diff
@@ -0,0 +1,19 @@
+GET /plone/front-page/@components/breadcrumbs
+Accept: application/json
+
+HTTP 200 OK
+content-type: application/json
+
+[
+  {
+    "data": {
```

Comment:
@ebrehault is there a specific reason why "items" is wrapped into "data"? Shouldn't we be consistent with our batching/folder/collection API here? cc @lukasgraf